tidb-external-ts.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Read Historical Data Using the `tidb_external_ts` Variable
+summary: Learn how to read historical data using the `tidb_external_ts` variable.
+---
+
+# Read Historical Data Using the `tidb_external_ts` Variable
+
+This document describes how to perform the [Stale Read](/stale-read.md) feature using the `tidb_external_ts` variable to read historical data in TiDB, including specific usage examples.
+
+> **Warning:**
+>
+> Currently, you cannot use Stale Read together with TiFlash. If your SQL query contains the `AS OF TIMESTAMP` clause and TiDB might read data from TiFlash replicas, you might encounter an error with a message like `ERROR 1105 (HY000): stale requests require tikv backend`.
```

Comment:
"If your SQL query contains the `AS OF TIMESTAMP`" maybe use should say "If you enables `tidb_enable_external_ts_read `"

Another thing is the warning is not in the Chinese document?

Reply:
Nice catch. Fixed.